gobblin-core/src/main/java/gobblin/metrics/GobblinMetrics.java

```diff
@@ -238,21 +237,20 @@ public void startMetricReporting(Properties properties) {
       return;
     }
 
-    buildFileMetricReporter(properties);
     long reportInterval = Long.parseLong(properties.getProperty(ConfigurationKeys.METRICS_REPORT_INTERVAL_KEY,
         ConfigurationKeys.DEFAULT_METRICS_REPORT_INTERVAL));
-    if (this.fileReporter.isPresent()) {
-      this.fileReporter.get().start(reportInterval, TimeUnit.MILLISECONDS);
-    }
 
     buildJmxMetricReporter(properties);
     if (this.jmxReporter.isPresent()) {
       this.jmxReporter.get().start();
     }
 
+    buildFileMetricReporter(properties);
     buildKafkaReporter(properties);
```

Comment:
Please rename this to `buildKafkaMetricReporter` to be consistent with the other two.